### DISCUSSION TITLE
How to delete plugin from DSA

### DISCUSSION THREAD

**Discussion by abhisheklomsh** [DigitalSlideArchive/digital_slide_archive#292] [Q&A] [answered]
As we upload plugin images from "Tasks / Slicer CLI Web Tasks" we are going to get drop-down under "analyses" tab, I want to know how to delete them.

To show the dropdown under "analyses" tab I am attaching a screenshot.
<img width="563" alt="Screenshot 2023-09-06 at 7 45 25 PM" src="https://github.com/DigitalSlideArchive/digital_slide_archive/assets/13444259/31307f64-76ec-4137-a555-2eb34d3b01f2">

**Comment by andreped** [comment, marked as answer]
In DSA, go to `Collections` > `Tasks` > `Slicer CLI Web Tasks` and then click on the plugin you wish to remove (e.g., `fastpathology`), and click on the folder icon top right, and press `Delete this folder`.

<img width="1677" alt="Screenshot 2023-09-07 at 08 45 30" src="https://github.com/andreped/FP-DSA-plugin/assets/29090665/7df6cb62-7aca-4911-a578-69255d20b657">

After removing it, `fastpathology` no longer shows up in the HistomicsUI:

<img width="944" alt="Screenshot 2023-09-07 at 08 47 54" src="https://github.com/DigitalSlideArchive/digital_slide_archive/assets/29090665/83bfd588-4b7e-42f5-bdcb-b008a91e6d44">

**Reply by abhisheklomsh** [reply]
Got it now, if I tick checkbox of anything be it file or folder, then these folder actions get disabled.

**Reply by manthey** [reply]
After checking items/folders in the list, the drop down for delete is next to the check/uncheck all box just above the check mark column.
